notebooks/official/workbench/subscriber_churn_prediction/telecom-subscriber-churn-prediction.ipynb

```diff
@@ -33,16 +33,17 @@
         "\n",
```

Comment:
<p>There is something wrong with the title. It is # UUID. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/874/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
I have cross checked. 

This is code for UUID

#### UUID

If you are in a live tutorial session, you might be using a shared test account or project. To avoid name collisions between users on resources created, you create a uuid for each instance session, and append it onto the name of resources you create in this tutorial.